content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -203,6 +203,5 @@ This section outlines the upgrade steps for Container Storage Modules (CSM) for
       - Update the driver config version and image tag
       - Update the Authorization config version and karavi-authorization-proxy image.
 
->__Note__: 
+>__Note__:
 > - In Authorization module upgrade, only `n-1` to `n` upgrade is supported, e.g. if the current observability version is `v1.8.x`, it can be upgraded to `1.9.x`.
```

Comment:
is this example right? Why are we referring observability here?

Reply:
fixed